terraform_compliance/common/terraform_files.py

```diff
@@ -28,7 +34,7 @@ def convert_terraform_plan_to_json(terraform_plan_file, terraform_executable=Non
     if terraform_executable is None:
         terraform_executable = which('terraform')
     else:
-        print('Using {} as terraform executable.'.format(terraform_executable))
+        print('. Using {} as terraform executable.'.format(terraform_executable))
 
     if terraform_executable is None:
         sys.stderr.write('ERROR: Could not find "terraform" executable in PATH. Please either use "-t" parameter '
```

Comment:
This will prevent downloading the correct terraform version if there are no terraform executables at all in the PATH. Is this intended?

Reply:
Yes it is. The plan is a `plan.out` but not `plan.out.json`, then there is no way for `terraform-compliance` to understand the `version` requirement of `terraform` that can convert that plan file into JSON.

Hence, if `plan.out` is given (not `plan.out.json`) and if there is no terraform in the `PATH` then this error must be given.